robot-command/src/main/java/org/obolibrary/robot/TemplateCommand.java

```diff
@@ -49,6 +53,7 @@ public TemplateCommand() {
         "A", "include-annotations", true, "if true, include ontology annotations from merge input");
     o.addOption("f", "force", true, "if true, do not exit on error");
     o.addOption("e", "errors", true, "write errors to this path (TSV or CSV)");
+    o.addOption("E", "ext-template", true, "external robot template data file");
```

Comment:
I'd prefer to call this `external-template` without abbreviating the name of the option.